runtime/test/src/generator/TemplateGeneratorTests.scala

```diff
@@ -0,0 +1,73 @@
+package swam
+package import_generator
+
+import java.io.File
+import java.nio.ByteBuffer
+import java.nio.file.Paths
+import java.util.concurrent.Executors
+
+import cats.effect.{Async, Blocker, IO}
+import pureconfig.ConfigSource
+import swam.runtime.{Engine, formats, imports}
+import swam.runtime.config.EngineConfiguration
+import swam.runtime.formats.DefaultFormatters
+import swam.runtime.import_generator.ImportGenerator
+import swam.runtime.imports.{AsInstance, AsInterface, TCMap}
+import swam.syntax.Section.Imports
+import swam.text.Compiler
+import swam.validation.Validator
+import utest.{TestSuite, Tests, test}
+
+import scala.concurrent.ExecutionContext
+import scala.reflect.runtime._
+
+/**
+  * @author Javier Cabrera-Arteaga on 2020-03-06
+  */
+object TemplateGeneratorTests extends TestSuite {
```

Comment:
These tests don't really look like tests to me, I would move this into example code instead.